composer.json

```diff
@@ -19,6 +19,7 @@
             "email": "ilja@scandiweb.com"
         }
     ],
+    "version": "2.6.0",
```

Comment:
do not commit version into composer.json

Reply:
Thanks! I will remove.